typescript/packages/common-runner/src/cell.ts

```diff
@@ -45,9 +50,10 @@ export interface Cell<T> {
   sink(callback: (value: T) => void): () => void;
   key<K extends keyof T>(valueKey: K): Cell<T[K]>;
   getAsProxy(path?: PropertyKey[], log?: ReactivityLog): CellProxy<T>;
-  getAsCell(): CellImpl<T>;
+  getAsCellReference(): CellReference;
+  toJSON(): { "/": string } | undefined;
```

Comment:
if `{ "/": string }` is the format for an id, should we have a type for this?

(also, I can ask claude but why is it this shape?  it seems awkward)

Reply:
It's how links to entities are encoded in IPLD: https://ipld.io/docs/codecs/known/dag-json/#:~:text=The%20only%20difference%20from%20regular%20JSON%20is%20that%20if%20you%20want%20to%20encode%20a%20link%2C%20you%20take%20the%20CID%2C%20encode%20it%20in%20base58%20(or%20base32%2C%20for%20CIDv0)%2C%20and%20flank%20it%20in%20a%20map%20with%20a%20%22/%22%20key%20%2D%2D%20so%20it%20looks%20like%20this%3A%20%7B%22/%22%3A%22Qmfoo%22%7D.